receiver/httpcheckreceiver/metadata.yaml

```diff
@@ -27,6 +27,9 @@ attributes:
   error.message:
     description: Error message recorded during check
     type: string
+  http.substring:
+    description: HTTP response substring of body
+    type: string
```

Comment:
Is this really important? This value will be constant based on configuration.

Reply:
yes, you are right, this value will be constant and i need to remove this attribute.

Reply:
Done